wagtailio/roadmap/templates/roadmap/roadmap_page.html

```diff
@@ -0,0 +1,83 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block content %}
+    <div class="grid">
+        <div class="hero hero--small-margin">
+            <div class="hero__inner">
+                <h1 class="hero__heading">{{ page.title }}</h1>
+            </div>
+        </div>
+    </div>
+
+    <div class="grid milestones">
+        <div class="milestones__border"></div>
+
+        {% for milestone in milestones %}
+            <section class="milestone" id="{{ milestone.display_title|slugify }}">
+
+                <div class="milestone__header">
+                    {% include "patterns/components/icon/icon.html" with icon="flag" classes="icon--relative" %}
+
+                    <h3 class="heading-three milestone__heading">
+                        <a class="milestone__heading-link" href="#{{ milestone.display_title|slugify }}">{{ milestone.display_title }}</a>
+                        {% if milestone.display_subtitle %}
+                            <span class="milestone__heading-subtitle">{{ milestone.display_subtitle }}</span>
+                        {% endif %}
+                    </h3>
+                    {% if perms.roadmap.change_milestone %}
+                        <a class="milestone__heading-link milestone__heading-link--edit" href="{{ milestone.get_admin_url }}">✏️</a>
+                    {% endif %}
+                </div>
+
+                <div class="milestone__items grid">
+                    {% for item in milestone.items.all %}
+                        <article class="milestone-item">
+                            <div>
+                                <a class="milestone-item__external-link" href="{{ item.url }}" target="_blank" rel="noreferrer">
+                                    <h4 class="milestone-item__heading teaser-intro">
+                                        <span>
+                                            {{ item.title }}
+                                        </span>
+                                        <svg class="icon" aria-hidden="true" viewBox="0 0 100 100">
+                                            <use xlink:href="#arrow-up-right-from-square">
+                                            </svg>
+                                        </h4>
+                                    </a>
+                                    <div class="milestone-item__labels">
+                                        {% for label in item.labels_list %}
+                                            <span class="milestone-item__label">{{ label }}</span>
+                                        {% endfor %}
+                                    </div>
+                                </div>
+                                {% if item.needs_sponsorship or item.sponsorship_url %}
+                                    {% pageurl page.sponsorship_page item.url as sponsor_url %}
```

Comment:
It's a fallback for the `pageurl` tag if the `sponsorship_page` foreign key is unset, so that it doesn't crash when it tries to compute from `None`. This `item.url` is only used if both the `sponsorship_page` FK on the page is unset and the item has no `sponsorship_url`. It's just for safeguarding more than anything.